functests/4_latency/latency.go

```diff
@@ -314,11 +320,12 @@ func getLatencyTestPod(profile *performancev2.PerformanceProfile, node *corev1.N
 		cpus := cpuset.MustParse(string(*profile.Spec.CPU.Isolated))
 		latencyTestCpus = cpus.Size() - 1
 	}
-
+	currentTime := time.Now().Format("20060102150405")
```

Comment:
do you need to know this file name? would a temporary file (see: https://pkg.go.dev/io/ioutil#TempFile) be good?

Reply:
hm we pass the name to the latency tool, so I think we do not really use temp files(because our wrapper is responsible to create the file)